vars/tdr.groovy

```diff
@@ -26,23 +26,23 @@ def runGitSecrets(repo) {
 //It is important for TDR devs to know that the code they want to merge doesn't break TDR. By sending the build status for every commit (all branches) to GitHub we can ensure code that breaks TDR cannot be merged.
 
 // Call this when build starts (to let person who made changes know they are being checked) - call within first 'stage' of Jenkins pipeline actions.
-def reportStartOfBuildToGitHub(String repo) {
+def reportStartOfBuildToGitHub(String repo, String sha) {
```

Comment:
This is a breaking change. How will you roll it out?

Give me a shout if you want to talk it through!